README.md

```diff
@@ -319,6 +319,12 @@ If there's a git mirror path and `mount-checkout` is enabled, the (mirror path)[
 
 Default: `true`
 
+### `chown` (optional, boolean)
+
+Whether to chown mounted files to the Buildkite agent user before exiting, to ensure that Buildkite can clean them up. Only useful if `mount-checkout` is enabled.
+
+Default: `false`
```

Comment:
I think this should default to `true`, since I can’t imagine wanting the prior behavior (which bricks the agent due to inability to e.g. `git clean`), but I made it `false` since it’s a behavior change. If the maintainers are ok with it, happy to change it to `true`, otherwise I recommend that it be flipped in the next major version.

Reply:
I agree `false` makes sense here for now; I think rewording:

> Has no effect on Windows or if `mount-checkout` is false.

to:


> If `mount-checkout` is false, or if using Windows, this option will do nothing.

(or something to that effect) will be help prevent confusion.